docs/api/records.rst

```diff
@@ -6,14 +6,16 @@ Records
 Records belong to a collection. It is the data being stored and
 synchronized.
 
-Sending a record
-================
+.. _records-post:
+
+Uploading a record
+==================
 
 .. http:post:: /buckets/(bucket_id)/collections/(collection_id)/records
 
-    **Requires authentication**
+    :synopsis: Stores a record in the collection, and its ID will be assigned automatically.
```

Comment:
The synopses in this document (`records.rst`) use a different voice than those in the other documents (that I've reviewed so far, at least). Consider the following:
> Stores a record in the collection.
vs.
Store a record in a collection.

The first statement is in an active voice ([imperative](https://en.wikipedia.org/wiki/Imperative_mood)) whereas the second is passive. It's a subtle distinction, and I'm not arguing for one or the other - only that we be consistent across the documentation.

In other words, the form of the synopses in this document should be changed to match that of the other documents. :smile: